miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/Type.kt

```diff
@@ -21,3 +22,9 @@ internal enum class AdType(val value: Int) {
     INTERSTITIAL(0),
     REWARDED(1),
 }
+
+internal enum class ScreenAction(val action: String) {
+    LOCK_PORTRAIT("rakuten.miniapp.screen.lock_portrait"),
+    LOCK_LANDSCAPE("rakuten.miniapp.screen.lock_landscape"),
+    LOCK_RELEASE("rakuten.miniapp.screen.lock_release"),
+}
```

Comment:
I think, we can handle any unspecified or unknown request as well. 
In that case, we can utilize `ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED`

Reply:
We should not do that because the miniapp can spam the requests to sdk, the change orientation on android takes resource to execute.